packages/apollo-server-caching/src/__mocks__/common.ts

```diff
@@ -0,0 +1,6 @@
+export const servers = {
```

Comment:
Having to deal with external dependencies means these are not really unit tests and we probably don't want to run them as part of CI for example. I think they are great as integration tests, so I'd like to keep them, but I'm not sure how we should organize them to separate them from the unit tests and what the best practices around integration testing are (@jbaxleyiii may have some ideas there).

A script to start a Memcached and Redis server ready for these tests would be useful I think. And maybe we could run it as part of a `npm run test:integration` command.

Reply:
I'm a lot less familiar with Redis (though I suspect it should be similarly easy) but Memcached should be incredibly easy to mock.  In fact, a quick search turned up [`memcached-mock`](https://www.npmjs.com/package/memcached-mock) and [`redis-mock`](https://www.npmjs.com/package/redis-mock).

If it's at all feasible to use those, rather than implementing a script to start one (which would need to struggle with cross-platform network/process implementation details, not to mention _installation_), that seems preferable.

Reply:
agreed! these integration tests probably don't need to be run with CI, its just for me to validate that the dependencies on external libraries work as expected. I'm running the databases with docker, will try to set it up as a npm script!

Reply:
update: turns out I cant get proxyquire to work with typescript. Using jest.mock instead and it works great!

Reply:
Even better :)